src/main/java/seedu/address/model/contact/ContactPhone.java

```diff
@@ -4,10 +4,10 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Contact's phone number in the address book.
+ * Represents a Contact's phone number in Teaching Assistant.
  * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
  */
-public class Phone {
+public class ContactPhone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
```

Comment:
#139 Are we following this person's suggestion?

Reply:
sure, will fix